packages/runtime/test/consumption/notifications.test.ts

```diff
@@ -122,12 +135,12 @@ describe("Notifications", () => {
             expect(result).toBeAnError("'LocalNotification' not found.", "error.transport.recordNotFound");
         });
 
-        test("wrong message id for sentNotification", async () => {
+        test("wrong message id for saveSentNotification", async () => {
             const result = await sConsumptionServices.notifications.sentNotification({ messageId: "wrong-id" });
             expect(result).toBeAnError("messageId must match pattern MSG\\[A-Za-z0-9\\]\\{17\\}", "error.runtime.validation.invalidPropertyValue");
         });
 
-        test("not existing message id for sentNotification", async () => {
+        test("not existing message id for saveSentNotification", async () => {
```

Comment:
- "Message ID" instead of "message id".
- Rephrase: "saveSentNotification" to "save Notification with status 'Sent'"